src/components/AccountModule/WalletSyncedInfo.js

```diff
@@ -150,4 +148,11 @@ function ConnectionInfoMessage({ connectionStatus, locator }) {
   )
 }
 
+ConnectionInfoMessage.propTypes = {
+  connectionStatus: PropTypes.oneOf([
+    'Symbol(STATUS_CLIENT_CONNECTION_DROPPED)',
```

Comment:
We should be able to use the symbol directly here, not its string representation.